geocoder_tester/base.py

```diff
@@ -134,23 +194,21 @@ def compare_values(get, expected):
 def assert_search(query, expected, limit=1,
                   comment=None, lang=None, center=None,
                   max_matches=None):
-    params = {"q": query, "limit": limit}
+    query_limit = CONFIG['CHECK_DUPLICATES'] or limit
```

Comment:
What if `limit` is larger that `CONFIG['CHECK_DUPLICATES']` ?  
Shouldn't we take the max, and use a different number of results for each check ?  
Both values have different roles, and I think setting `CONFIG['CHECK_DUPLICATES']` shouldn't change the number of `SearchException` because more (or less) results would be fetched.